### PR TITLE
stack: add page

### PR DESCRIPTION
**Pull request by mstruebing**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines 

If interested to read about stack: https://docs.haskellstack.org/en/stable/README/

### CODE REVIEW

**Comment by sbrl** [comment]
Awesome, thanks @mstruebing! Looks like @tldr-bot detected a few issues though.

**Comment by mstruebing** [comment]
Thank you @sbrl . Wasn't it just one issue which I already fixed? :)

File: pages/common/stack.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+- Compile a project and re-compile every time a file changes:
+
+`stack build --file-watch`
+
+- Compile a project and execute a command after compilation:
+
+`stack build --exec "{{command}}"`
+
+- Compile a project and re-compile every time a file changes and execute a command after compilation:
```
Comment (sbrl) [inline pages/common/stack.md]: I'm not sure this one is needed since you've already demonstrated each function individually above.

Comment (mstruebing) [reply pages/common/stack.md]: I wanted to demonstrate that you can mix these parameters as you want.
I will remove it :+1:

Comment (sbrl) [reply pages/common/stack.md]: I think it's something that the average user should be able to figure out :-)

@agnivade / @waldyrious thoughts?

Comment (agnivade) [reply pages/common/stack.md]: Less is good. But not at the cost of clarity. In this case, I think it's fine.

Comment (mstruebing) [reply pages/common/stack.md]: Does that mean I should re-add this example?

Comment (agnivade) [reply pages/common/stack.md]: Nope. It's fine. i.e. keep it removed.

**Comment by waldyrious** [comment]
LGTM. Thanks for the contribution, @mstruebing!

**Pull request merged**